src/plugins/album-actions/index.tsx

```diff
@@ -52,19 +57,69 @@ export default createPlugin<
       }
       const continuations = await waitForElement<HTMLElement>('#continuations');
       this.waiting = false;
-      //Gets the for buttons
-      const buttons: Array<HTMLElement> = [
-        ElementFromHtml(undislikeHTML),
-        ElementFromHtml(dislikeHTML),
-        ElementFromHtml(likeHTML),
-        ElementFromHtml(unlikeHTML),
-      ];
+
+      const [showUnDislike, setShowUnDislike] = createSignal(true);
+      const [showDislike, setShowDislike] = createSignal(true);
+      const [showLike, setShowLike] = createSignal(true);
+      const [showUnLike, setShowUnLike] = createSignal(true);
+
+      const DEFAULT_MASK_SIZE = '100% 50%';
+      const [unDislikeMaskSize, setUnDislikeMaskSize] =
+        createSignal(DEFAULT_MASK_SIZE);
+      const [dislikeMaskSize, setDislikeMaskSize] =
+        createSignal(DEFAULT_MASK_SIZE);
+      const [likeMaskSize, setLikeMaskSize] = createSignal(DEFAULT_MASK_SIZE);
+      const [unLikeMaskSize, setUnLikeMaskSize] =
+        createSignal(DEFAULT_MASK_SIZE);
+
+      const buttonContainer = document.createElement('div');
+      buttonContainer.style.display = 'flex';
+      buttonContainer.style.flexDirection = 'row';
+
+      render(
+        () => (
+          <>
+            <Show when={showUnDislike()}>
+              <UnDislikeButton
+                onClick={this.loadFullList}
+                maskSize={unDislikeMaskSize()}
+              />
+            </Show>
+            <Show when={showDislike()}>
+              <DislikeButton
+                onClick={this.loadFullList}
+                maskSize={dislikeMaskSize()}
+              />
+            </Show>
+            <Show when={showLike()}>
+              <LikeButton
+                onClick={this.loadFullList}
+                maskSize={likeMaskSize()}
+              />
+            </Show>
+            <Show when={showUnLike()}>
+              <UnLikeButton
+                onClick={this.loadFullList}
+                maskSize={unLikeMaskSize()}
+              />
+            </Show>
+          </>
```

Comment:
```suggestion
          <Switch>
            <Match when={showUnDislike()}>
              <UnDislikeButton
                onClick={this.loadFullList}
                maskSize={unDislikeMaskSize()}
              />
            </Match>
            <Match when={showDislike()}>
              <DislikeButton
                onClick={this.loadFullList}
                maskSize={dislikeMaskSize()}
              />
            </Match>
            <Match when={showLike()}>
              <LikeButton
                onClick={this.loadFullList}
                maskSize={likeMaskSize()}
              />
            </Match>
            <Match when={showUnLike()}>
              <UnLikeButton
                onClick={this.loadFullList}
                maskSize={unLikeMaskSize()}
              />
            </Match>
          </Switch>
```

DO NOT FORGET adding `Switch` and `Match` imports!